Feature/UI 3

gemansipator opened the pull request:
На проверку))

gemansipator committed:
Начал создавать компонент, также изменил меню

gemansipator committed:
отобразил результат на главной

gemansipator committed:
создал вторую новость с большим текстом

gemansipator committed:
CardBaseTwo

gemansipator committed:
CardBaseTwo

gemansipator committed:
Почемуто не пушилось Перезалил код.

gemansipator committed:
вывел СТАТИЧНУЮ картинку не через json.....

gemansipator committed:
картинку из json из интернета загружается
а локальные нет

gemansipator committed:
добавил маркеры и текст

gemansipator committed:
картинку из json из интернета загружается
а локальные нет

gemansipator committed:
таблица, чекбоксы, шестеренка

gemansipator committed:
таблица, чекбоксы, шестеренка

gemansipator committed:
таблица, чекбоксы, шестеренка

Lekso4ka commented:
По задачам UI-3 / UI-7

Lekso4ka commented on src/components/table/Table.jsx:
```diff
@@ -0,0 +1,127 @@
+                    </div>
+                </td>
+                <td>
+                    <button className="image-button"><img src={Gear} alt={"картинка"}/></button>
```
import {SettingOutlined} from @ant-design/icons

....

<.  SettingOutlined     />

gemansipator committed:
В файле assets/data/news.json поменять структуру данных на следующую:
У каждой новости должны быть свойства:

"title",
"titleImage",
"text",
"forStudents": Boolean (true/false),
"forTeachers": Boolean (true/false),
"createdAt": Date ("2023-10-01")

gemansipator committed:
Наполнить массив минимум 5 новостями (реальными), из них хотя бы одна новость должна быть для преподавателей и для студентов

gemansipator committed:
На странце с новостями отобразить таблицу со всеми новостями, в которой будут отображаться следующие столбцы:
- Название
- Дата создания
- Текст
- Видимость для студентов НЕ СДЕЛАНО
- Видимость для преподавателей НЕ СДЕЛАНО
- Управление (кнопки для дальнейшего изменения новости)

gemansipator committed:
Ошибка при попытке прописать map в шаблоне (компоненте) Table

gemansipator committed:
- Видимость для студентов (в поле поставить маркер, если новость видна пользователям)
- Видимость для преподавателей (в поле поставить маркер, если новость видна преподавателям)
